website/docs/export.md

```diff
@@ -209,6 +211,121 @@ Dapr: https://dapr.io/
     2023/07/18 20:37:20 main.ExportMsg{ID:"2023-07-18T20:37:19Z", Details:map[string]interface {}{"missing_labels":[]interface {}{"test"}}, EventType:"violation_audited", Group:"constraints.gatekeeper.sh", Version:"v1beta1", Kind:"K8sRequiredLabels", Name:"pod-must-have-test", Namespace:"", Message:"you must provide labels: {\"test\"}", EnforcementAction:"deny", ConstraintAnnotations:map[string]string(nil), ResourceGroup:"", ResourceAPIVersion:"v1", ResourceKind:"Pod", ResourceNamespace:"nginx", ResourceName:"nginx-deployment-58899467f5-j85bs", ResourceLabels:map[string]string{"app":"nginx", "owner":"admin", "pod-template-hash":"58899467f5"}}
     ```
 
+### Quick start with exporting violations on node storage using Disk driver via emptyDir
+
+#### Prerequisites for Disk driver
+
+1. Build `fake-reader` image from [gatekeeper/test/export/fake-reader](https://github.com/open-policy-agent/gatekeeper/tree/master/test/export/fake-reader)
+
+    ```bash
+    docker buildx build -t <your_img_name:tag> --load -f test/export/fake-reader/Dockerfile test/export/fake-reader
+    ```
+
+2. Update `gatekeeper-audit` deployment to add `emptyDir` volume.
+
+    ```yaml
+    volumes:
```

Comment:
do we want to add this to helm chart with conditional?

Reply:
I found it to be more complecated to add a whole sidecar container and its config because unlike adding dapr annotation which adds a container with specification required for dapr sidecar to run. A sidecar reader for purposes beyond just reading and logging would require specific customizations. We can add it but the chart will get more complicated and I do not see the value since we might not be able to make it completly customizable.

Reply:
Updated to add helm variables for `volume`, `volumeMount` and `sidecar` as well.